docs/language/operators.md

```diff
@@ -17,7 +17,6 @@ The following available operators are documented in detail below:
 * [`head`](#head)
 * [`join`](#join)
 * [`over`](#over)
-* [`pick`](#pick)
```

Comment:
How about a doc for `let`?

Reply:
Or actually it doesn't look like its actually implemented...

Reply:
It's implemented up to semantic, but not it appears in to be in the kernel (perhaps I am missing something?). If not, maybe get rid of this change- it'll make it easier to review it when it goes in.

Reply:
I'm trying to move fast rather than make the PR perfect.  The motivation for this PR was Let and scopes, so I put a few things in as scaffolding.  This is mentioned in the PR description and git comment.